Project/backend/codebase/graph_creator/llama3.py

```diff
@@ -0,0 +1,112 @@
+import os
+import json
+from datetime import datetime
+from dotenv import load_dotenv
+from groq import Groq
+
+def configure_groq():
+    """
+    Ensure the API key is set in the environment (set it in the .env file or Linux/Mac: export GROQ_API_KEY="Your_API_KEY")
+    """
+    load_dotenv("Project/backend/.env", override=True)
```

Comment:
This line needs to be removed since docker already loads the `.env` variables.
Please update also the `.env.example` with the new needed variables